### PR TITLE
[RDY] implementation of transaction test

### PR DESCRIPTION
**Pull request by jangko**

### CODE REVIEW

**Comment by jangko** [comment]
blocking issue from status-im/nim-eth#95

**Comment by jangko** [comment]
this PR is ready

File: nimbus/transaction.nim [hunk 1]
```diff
@@ -19,18 +20,14 @@ func intrinsicGas*(data: openarray[byte]): GasInt =
     else:
       result += 68  # GasTXDataNonZero
 
-proc intrinsicGas*(t: Transaction): GasInt =
+proc intrinsicGas*(tx: Transaction, fork: Fork): GasInt =
```
Comment (stefantalpalaru) [inline nimbus/transaction.nim]: The `fork` parameter will be used in the future?

Comment (jangko) [reply nimbus/transaction.nim]: it's already been used in `validate` proc

**Pull request merged**